inst/css/BS4/pkgdown.css

```diff
@@ -413,11 +397,11 @@ mark {
 }
 
 .algolia-autocomplete .aa-hint {
-  color: #999;
+  color: mix($black, $white, 10%);
```

Comment:
Do we still need the algolia styling?

Reply:
Yes, because this comes from algolia autocomplete that we still use.